package.json

```diff
@@ -8,11 +8,17 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
+    "test": "jest --no-watchman",
     "clean": "rm -rf dist tsconfig.tsbuildinfo",
     "prepublishOnly": "yarn clean && yarn build"
   },
   "dependencies": {
+    "@ethereumjs/util": "^9.0.0",
```

Comment:
I see we only use `bytesToHex` from this package. Isn't a similar method available in `ethers`? Would avoid adding a new package just for a single basic method.